### DISCUSSION TITLE
Authorization via clientjstoken

### DISCUSSION THREAD

**Discussion by wbafy** [SteamRE/SteamKit#1394] [General]
### What problem is this feature trying to solve?

The steam libraries in python and nodejs already have such a function.

### How would you like it to be solved?

You need to add clienjstoken to SteamUser.LoginOn so that you can log in to the chat using it the same way. This can greatly simplify your work.

### Have you considered any alternative solutions

Access SteamСhat directly using Clientjstoken.

### Additional Information

I'm just learning programming and I apologize if I write something wrong

**Comment by WabiSabikk** [comment]
+
